### DISCUSSION TITLE
How to limit revisions bin by age

### DISCUSSION THREAD

**Discussion by bwmeyer** [ravendb/ravendb#17305] [Q&A]
I have configured revisions for some collections.
They are limited to max 5 revisions without a limit in age.
Also for deleted documents the revisions will not be purged.

As I understand, with this configuration, the revisions bin will grow infinitely.
So I want to limit the revisions bin.
Unfortunately, I cannot find any documentation about this online.

How can I achieve to limit the revisions hold in the revisions bin by age (related to the deletion time) without affecting the revisions of currently available documents?

**Comment by shaharhikri** [comment]
We opened an issue for that.
You can check its progress here:
https://issues.hibernatingrhinos.com/issue/RavenDB-21381/Add-EP-for-adding-Delete-Revision-to-orphaned-revisions-and-an-additional-expiration-time-for-deleting-deleted-doc-revisions.
